integration-tests/exporter/exporter_test.go

```diff
@@ -0,0 +1,242 @@
+// Copyright © 2019 The OpenEBS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/integration-tests/artifacts"
+	framework "github.com/openebs/maya/integration-tests/framework/v1alpha1"
+	k8s "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	cv "github.com/openebs/maya/pkg/cstorvolume/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// namespaceYaml holds the yaml spec
+	// to create admission namespace
+	namespaceYaml artifacts.Artifact = `
```

Comment:
We shall make use of Builder to build every resource & each resource's KubeClient to trigger Kubernetes API calls